data/Canham_2009/metadata.yml

```diff
@@ -0,0 +1,153 @@
+    a pressure collar, stem ends were fitted to a steady-state flow meter (SSFM).
+    The SSFM uses a pressure drop across PeekTM capillary tubing (which was maintained
+    at less than 5kPa) of a known diameter ordered in series with the stem segment.
+    An initial maximum measurement of hydraulic conductivity kh (kg m s^-1 MPa^-1)
+    (Patino, Tyree & Herre 1995) was made after flushing the stem with a filtered
+    (0.11µm) degassed perfusion solution (1 mm KCl in double-distilled water). Air
+    was then injected into the collar to a desired pressure and maintained for 15
+    min. The air injection pressure is equivalent to the water potential of xylem
+    multiplied by negative one. The pressure in the collar was then slowly released
+    and kh was remeasured. This procedure was repeated to progressively higher pressures
+    (ranging from 0.0 to 5.0 MPa). To describe the relationship between PLC and Yx
+    exponential sigmoidal functions (Pammenter & Vander Willigen 1998) were fitted
+    to the data.
+- var_in: Ks (kg/m2/Mpa/s)
+  unit_in: kg/m/MPa/s
+  trait_name: stem_specific_hydraulic_conductivity
+  entity_type: individual
+  value_type: raw
+  basis_of_value: measurement
+  replicates: 1
+  methods: Using the same apparatus as described for vulnerability curves, maximum
+    hydraulic conductivity of stem xylem, normalized to stem cross-sectional area
+    (Ks; in kg s^-1 m^-1 MPa^-1), was measured in stem segments according to the general
+    principle of Zimmermann (1978).Ks is defined by KS= Jv x <\u0394>l / <\u0394>P,
+    where Jv (kg(water)^s-1 m^-2(stem)) is the mass flux density of the perfusion
+    solution (0.01m KCl in degassed, double-distilled water), <\u0394>l is the length
+    of the stem segment (m) and <\u0394>P is the water pressure difference along the
+    length of stem segment (MPa). Measurements of Ks were made after removing embolisms;
+    hence the values reported could be described as maximum hydraulic conductivity.
+- var_in: Kl (kg/m2/Mpa/s)
+  unit_in: kg/m/MPa/s
+  trait_name: leaf_specific_hydraulic_conductivity
+  entity_type: individual
+  value_type: raw
+  basis_of_value: measurement
+  replicates: 1
+  methods: Leaf specific hydraulic conductivity (KL), defined here as the mass flow
+    movement of water through the stem per unit of pressure gradient per unit of leaf
+    area, was determined as KL = KS x H (kg s^-1 m^-1 MPa^-1), where H is the Huber
+    value, the ratio of stem cross sectional area Ax (m2) to leaf area AL(m2) receiving
+    water from that stem. Leaf area was measured with an area meter (model Delta-T
+    Type WDIGC-2, Delta T Devices, Cambridge UK) and Ax was measured accurately with
+    digital callipers.
+- var_in: H
+  unit_in: m2{sapwood}/m2{leaf}
+  trait_name: huber_value
+  entity_type: individual
+  value_type: raw
+  basis_of_value: measurement
+  replicates: 1
+  methods: Huber value, the ratio of stem cross sectional area Ax(m2) to leaf area
```

Comment:
Not usually, because Huber value is calculated on a branch that is still 100% sapwood. There will be a thin bark layer, but I assume they aren't including that.